RELEASE_NOTES.md

```diff
@@ -122,13 +125,17 @@
 * Fixes bug where default input values were not overridden for task inside subworkflow
 * Adds support for specifying native app(let) in `runtime` section
 * Fixes some type conversion bugs related to CWL `Any` type
+* Affected by a bug for workflows that include a native platform app via `dxni`. Bug fixed in v2.8.1.
+* Logs entire contents of WDL command at runtime
```

Comment:
Adding back some release notes that I saw on the release page that weren't on develop branch.